src/averageAges.js

```diff
@@ -37,7 +73,44 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womans = people.reduce((allWomans, woman) => {
+    woman.sex === 'f' && allWomans.push(woman);
```

Comment:
'f' is a mafic string here. See my previous comment

There is a repeating code that manages the sex check. Consider using a separate function isWoman or isMan. It should return true or false accordingly.